src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольник
+
+    public Rectangle(String color,double firsSide, double secondSide) {
+        super(color, firsSide, secondSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return getFirsSide() * getSecondSide();
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Длина = " + getFirsSide()
+                + " Ширина = " + getSecondSide()
+                + " Площадь прямоугольника = " + detAreaOfFigure());
```

Comment:
Avoid using non-English text in your code. The checklist advises using only English in messages and code to maintain universality and consistency.